server/routes/statistics.js

```diff
@@ -120,6 +121,20 @@ var businessProcessQueryHandler = new QueryHandler([{
 	}
 }]);
 
+var getPeriods = function () {
+	var periods = ['inPeriod', 'today', 'thisWeek', 'thisMonth']
+	  , today = toDateInTz(new Date(), db.timeZone)
+	  , currentYear = new db.Date(today.getUTCFullYear(), 0, 1)
+	  , lastYearInRange = new db.Date(today.getUTCFullYear() - 5, 0, 1);
```

Comment:
It will be outdated in less than a year. I think it's better to hardcode that one in. This would mean changing `view/statistics-dashboard.js` as well. I understand that we want to show all years from beginning of the system, not only last 5 years. Maybe ask Vianney about it.